app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -364,12 +487,19 @@ class OptionsFragmentTest {
         )
       onView(isRoot()).perform(orientationLandscape())
       onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
-      onView(atPositionOnView(R.id.options_recyclerview, 2, R.id.audio_language_text_view)).check(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          2, R.id.audio_language_text_view
+        )
+      ).check(
         matches(withText("Hindi"))
       )
     }
   }
+  /* ktlint-enable max-line-length */
```

Comment:
Ditto

Reply:
Done

Reply:
Resolving this on behalf of @vinitamurthi . Though it was not mentioned by her, but as the changes are are fairly simple and this PR has been approved below by @vinitamurthi.